### DISCUSSION TITLE
Raycast Extension

### DISCUSSION THREAD

**Discussion by NoumanAMalik** [anyproto/contributors#84] [Anytype apps]
Hey guys, 
Just like how there is a Notion extension inside Raycast, would it be cool to make a Anytype extension for Raycast. I would be glad to work on it, as I think for productivity on Macs it would be very nice.

**Comment by fuksman** [comment]
Hey @NoumanAMalik, thank you for the suggestion. We also dream about a Raycast extension for Anytype. It will become possible when we launch our first automation capabilities later this year.

**Reply by NoumanAMalik** [reply]
Sweet! I'll keep an eye out for when that releases and then I'll try to work on the extension.

**Reply by captorxxv** [reply]
Any News On that ??

**Comment by jmetrikat** [comment]
@NoumanAMalik @captorxxv Good news! The Raycast extension for Anytype is finally live 🎉 

Grab it here: https://www.raycast.com/any/anytype or check out the repo: https://github.com/anyproto/anytype-raycast
